### PR TITLE
Add tool version and info fields

### PR DESCRIPTION
**Pull request by anthturner**
Signed-off-by: Anthony Turner <225599+anthturner@users.noreply.github.com>

**- What I did**
Added fields `SecurityToolVersion` and `SecurityToolInfoURI` to `vulnerability` object

**- How to verify it**
Add information to fields and create JSON export

**- Description for the changelog**
Added version and informational URI to security tool report

### CODE REVIEW

**Comment by anthturner** [comment]
This is related to https://github.com/ZupIT/horusec/pull/946#discussion_r794646046

File: pkg/entities/vulnerability/vulnerability.go [hunk 1]
```diff
@@ -60,6 +60,9 @@ type Vulnerability struct {
 	//
 	// For more info see https://github.com/ZupIT/horusec/issues/680
 	VulnHashInvalid string `json:"-" gorm:"-" swaggerignore:"true"`
+
+	SecurityToolVersion string `json:"securityToolVersion" gorm:"Column:security_tool_version" example:"1.2.0"`
+	SecurityToolInfoURI string `json:"securityToolInfoUri" gorm:"Column:security_tool_info_uri" example:"https://horusec.io/"`
```
Comment (matheusalcantarazup) [inline pkg/entities/vulnerability/vulnerability.go:65]: I think that we should ignore this field on Gorm. What do you guys think? @nathanmartinszup @wiliansilvazup @iancardosozup ?

Comment (nathanmartinszup) [reply pkg/entities/vulnerability/vulnerability.go:65]: I agree with @matheusalcantarazup.

**Review by matheusalcantarazup — APPROVED** [review]
Thanks very much for your contribution @anthturner

**Review by nathanmartinszup — APPROVED** [review]

**Pull request merged**